api/Worklet.json

```diff
@@ -66,6 +66,45 @@
             "deprecated": false
           }
         }
+      },
+      "ecmascript_modules": {
+        "__compat": {
+          "description": "Support for ECMAScript modules",
+          "support": {
+            "chrome": {
+              "version_added": "80"
+            },
+            "chrome_android": "mirror",
+            "deno": {
+              "version_added": "1.0"
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "114"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "15",
+              "notes": [
+                "Nested workers support was introduced in Safari 15.5.",
+                "Script loading in nested workers was introduced in Safari 16.4."
+              ]
```

Comment:
This is copied from the Safari Worker support info. I know this is supported in audio worklets now, I know that it is supported in the same release as worker support based on the names of the loaders in https://bugs.webkit.org/show_bug.cgi?id=164860

So I pretty much believe it will have same kinds of restrictions as for other workers.